src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx

```diff
@@ -171,8 +171,6 @@ The following attributes are required in the APM service entity to display the K
 * `k8s.namespace.name`
 
 <Callout variant="caution">
-The Kubernetes summary page won't work properly if there is a mix of different instrumentation providers (New Relic and OpenTelemetry). For it to work properly, both Kubernetes and APM need to be monitored either exclusively through New Relic's proprietary agents or entirely through OpenTelemetry. 
-
-**Support for different instrumentation providers setup is expected by 2025 Q1.**
+The Kubernetes summary page won't work properly if Kubernetes is OpenTelemetry and APM is New Relic.
```

Comment:
If I'm correct, installing nri-metadata-injection will provide the needed K8s metadata to the APM metrics.
Should we propose it as a possible solution?